modules/serverless-rn-1-11-0.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-11-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.11.0
+
+[id="new-features-1-11-0_{context}"]
+== New features
+
+* Knative Eventing on {ServerlessProductName} is now GA.
+* {ServerlessProductName} Functions is now delivered as a Developer Preview through the standard Knative `kn` CLI installation. This feature provides CLI commands that developers can use to write functions using predefined runtimes and languages, including Quarkus, Node.js and Go. Functions can then be triggered using CloudEvents and plain HTTP requests.
++
+This feature is not yet supported by Red Hat for production deployments, but can be used for development and testing purposes. For more information about using {ServerlessProductName} Functions through the `kn func` CLI, see the link:https://openshift-knative.github.io/docs/docs/functions/about-functions.html[{ServerlessProductName} Functions Developer Preview] documentation.
+* {ServerlessProductName} now uses Knative Operator 0.17.0.
+* {ServerlessProductName} now uses Knative Serving 0.17.0.
+* {ServerlessProductName} uses Knative Eventing 0.17.0.
+* {ServerlessProductName} now uses Kourier 0.17.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.17.1.
+// check versions
```

Comment:
Regarding the versions above. You can take this file as the source of truth: https://gitlab.cee.redhat.com/wvanwinc/productize-svls/-/blob/release-1.11/config.yaml (and for future releases just change the branch to e.g. release-1.12)
I can see the following versions there: Serving 0.17.3, Eventing 0.17.2, CLI 0.17.3, Kourier 0.17.0. 
The Knative Operator is there as "serverlessOperator": https://gitlab.cee.redhat.com/wvanwinc/productize-svls/-/blob/release-1.11/config.yaml#L41 , the version is 1.11.0.
However, I'm not sure if it makes sense to mention this operator anymore. It doesn't match any exact version from upstream and we started versioning it according to the product version. @markusthoemmes @warrenvw wdyt ?

Reply:
Thanks @mgencur !

Reply:
Not sure if we even have to mention it :man_shrugging:. It's an implementation detail of the product, so I wouldn't bother too much.

Reply:
Yeah. I agree. Let's remove that. It's just confusing.